src/Lifecycle.js

```diff
@@ -344,6 +347,9 @@ async function _doSetLoggedIn(credentials, clearStorage) {
                     localStorage.setItem("mx_team_token", body.team_token);
                 }
                 return body.team_token;
+            }, (err) => {
```

Comment:
yes, because we return null from this bit, which means the `then` handler at line 356 is called.